### DISCUSSION TITLE
What happened to the New LTMeat/LTMeat Flood exploit

### DISCUSSION THREAD

**Discussion by DankFireR** [3kh0/ext-remover#823] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
title says it all

**Comment by AshtonDavies** [comment, marked as answer]
He thought it was useless but will supposedly bring it back. You can still view the instructions [here](https://chrose.netlify.app/detail/LTMEAT%20Flood)

**Reply by DankFireR** [reply]
ah thank you
it isn't useless to me its how i bypassed securly lol

**Comment by 3kh0** [comment]
Adding it now

**Reply by DankFireR** [reply]
thank you
